setup.py

```diff
@@ -1,8 +1,8 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='PyDreamScreen',
-    version='0.0.6',
+    name='PyDreamScreen.HA',
```

Comment:
Revert to regular name:

```suggestion
    name='PyDreamScreen',
```